<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_log">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>                 登录</string>
   </property>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_pw">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
